pom.xml

```diff
@@ -161,8 +161,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.12.1</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.7</source>
+          <target>1.7</target>
```

Comment:
I'm OK increasing the version since we will need to further increase it to address #155.  But this is not strictly necessary it just requires compiling with an older JDK as the CI does.  gson itself dropped Java 6 compatibility in 2.9.0:

https://github.com/google/gson?tab=readme-ov-file#minimum-java-version

Reply:
Either way works. Using a more modern java version would make more sense to me as modernizer is all about being modern.